sencha-workspace/SlateCSV/app/view/Importer.js

```diff
@@ -1,9 +1,11 @@
-/*jslint browser: true, undef: true, laxcomma:true *//*global Ext*/
+/*jslint browser: true, undef: true, plusplus: true */ /*global Ext*/
```

Comment:
really? Who the hell doesn't use ++ in a for loop. What editor/lister are you using? I've never seen one complain about those out of the box